pi-apps-terminal-bash-edition.sh

```diff
@@ -63,80 +53,109 @@ function about() {
     '
 }
 
+function list_apps() { # $1 can be: installed, uninstalled, corrupted, cpu_installable, hidden, visible, online, online_only, local, local_only
+  if [ -z "$1" ] || [ "$1" == local ];then
```

Comment:
```suggestion
  if [[ -z "$1" ]] || [[ "$1" == "local" ]]; then
```

this comment and bellow are just to use my preferred coding style.